File: java_backup/my java/souvaghya/palindrome.java
class palindrome
{
public static void main(int n)
{
int n1=0,n2=n;
while(n>0)
{
n1=(n1*10)+n%10;
n=n/10;
}
if(n1==n2)
System.out.print("palindrome");
else
System.out.print(" non-palindrome");
}
}

File: 4rth_Sem_c++_backup/SHIT/palindrome.cpp
#include"iostream.h"
#include<conio.h>
int main()
{
    //clrscr();
    int n,i,j,k,n1,sum=0;
    cin>>n;
    sum=0;n1=n;
    while(n>0)
    {
      k=n%10;
      sum=(sum*10)+k;
      n=n/10;
    }
      cout<<n<<endl;
      if(n1==sum)
           cout<<"palindrome"<<endl;
      else
      cout<<"not a palindrome"<<endl;
      getch();
      return 0;
}        
